helm/squidex/values.yaml

```diff
@@ -46,6 +46,12 @@ affinity: { }
 ## @skip clusterSuffix
 clusterSuffix: cluster.local
 
+## @skip podSecurityContext
+podSecurityContext: { }
+
+## @skip containerSecurityContext
+containerSecurityContext: { }
```

Comment:
An example of a **Container Security Context** that follows the rules defined by the strictest policy outlined by Kubernetes (**Restricted**):

```
containerSecurityContext:
  capabilities:
    drop:
      - ALL
    add:
      - NET_BIND_SERVICE
  allowPrivilegeEscalation: false
```